python_ta/contracts/__init__.py

```diff
@@ -321,17 +331,71 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
 
 
 def check_type_strict(argname: str, value: Any, expected_type: type) -> None:
-    """Ensure that ``value`` matches ``expected_type``.
+    """
+    Ensure that `value` matches ``expected_type`` with strict type checking.
```

Comment:
Descriptions should always start on the same line as the triple-quotes (you should not change this). Make sure the functions you've added start with a short description after the triple-quotes.